legacysms/models.py

```diff
@@ -1,3 +1,29 @@
-# from django.db import models
+from django.db import models
 
-# Create your models here.
+
+class MediaItem(models.Model):
+    #: SMS media id
+    id = models.BigIntegerField(primary_key=True, editable=False, help_text='Legacy SMS media id')
+
+    #: Corresponding :py:class:`mediaplatform.MediaItem`. Accessible from the
+    #: :py:class:`mediaplatform.MediaItem` model via the ``sms`` field.
+    item = models.OneToOneField('mediaplatform.MediaItem', related_name='sms',
```

Comment:
currently we can't delete videos from the SMS via this method, models.CASCADE will make it disappear form the website but not from the SMS. It's a valid option but we need to think if that then will break our sync and it will try it to upload it again

Reply:
At no point will the UMP become the source of truth for the SMS so at no point will deleting a video from UMP be expected to delete it from the SMS. Rather, when the SMS ceases to be the source of truth, a video deleted on the UMP will remain on the SMS because they are unconnected systems. While the SMS is still the source of truth for the UMP, "deleting a video" should be done on the SMS and then synchronised to the UMP.

As I understand it, therefore, this is working as designed.